ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -100,6 +107,9 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
                     <Route path="/" exact render={() => <Redirect to={dashboardPath} />} />
                     <Route path={mainPath} exact render={() => <Redirect to={dashboardPath} />} />
                     <Route path={dashboardPath} component={AsyncDashboardPage} />
+                    {isDashboardPatternFlyEnabled && (
```

Comment:
Although this follows recent pattern, here is a though about pro and con of alternative patterns:
* Alternative path helps compare and contrast classic and PatternFly behavior on the fly.
* If that is not needed, then either/or rendering provides more realistic behavior at the edges:
    * Visit PatternFly page from left nav (including in new integration tests).
    * Redirect to PatternFly page at /main/dashboard path from partial / and /main paths
    * A con which we have precluded is any conflicting sagas at /main/dashboard

Reply:
I think the biggest driver for this pattern is that it will allow us to run e2e tests against the current dashboard as well as new e2e tests on the new dashboard in CI, throughout development and regardless of the state of the feature flag in release builds. More realistic test behavior is definitely a plus, but I think should (hopefully) be a minimal part of the test that we can work without until we enable the flag by default.

On a side note, I think the final saga that triggers at /main/dashboard was removed so we don't have to worry about that as a con. 😄

Reply:
Good point about both integration tests. I agree.